index-singlecell.md

```diff
@@ -0,0 +1,126 @@
+BEDTools | Utilities for genome arithmetic | [Quinlan and Hall 2010](https://doi.org/10.1093/bioinformatics/btq033){:target="_blank"}
+deepTools | Tools for exploring deep-sequencing data | [Ramirez et al. 2014](https://doi.org/10.1093/nar/gku365){:target="_blank"}, [Ramirez et al. 2016](https://doi.org/10.1093/nar/gkw257){:target="_blank"}
+UMI-tools | Modeling sequencing errors in Unique Molecular Identifiers to improve quantification accuracy | [Smith et al. 2017](https://doi.org/10.1101/gr.209601.116){:target="_blank"}
+Je | A versatile suite to handle multiplexed NGS libraries with unique molecular identifiers | [Girardot at al. 2016](https://doi.org/10.1186/s12859-016-1284-2)
+[RaceID]({{ page.website }}/?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/raceid_clustering/raceid_clustering/) | Clustering using RaceID performs clustering, outlier detection, dimensional reduction | [Josip, at al. 2018](https://doi.org/10.1038/nmeth.4662)
+{: .table.table-striped .tooltable}
+
+
+# Training
+
+We are passionate about training. So we are working in close collaboration with the [Galaxy Training Network (GTN)](https://galaxyproject.org/teach/gtn/){:target="_blank"} to develop training materials of data analyses based on Galaxy {% cite batut2017community %}. These materials hosted on the GTN GitHub repository are available online at [https://training.galaxyproject.org](https://training.galaxyproject.org){:target="_blank"}.
+
+We developed [several tutorials](https://galaxyproject.github.io/training-material/topics/transcriptomics/){:target="_blank"} specifically for scRNA processing and more will come:
+
+- [Introduction to Transcriptomics](https://galaxyproject.github.io/training-material/topics/transcriptomics/slides/introduction.html){:target="_blank"}
+- [Plates, Batches, and Barcodes](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-plates-batches-barcodes/slides.html){:target="_blank"}
+- [Understanding Barcodes](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-umis/tutorial.html){:target="_blank"}
+- [Pre-processing of Single-Cell RNA Data](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.html){:target="_blank"}
+
+
+# Contributors
+
+- [Bérénice Batut](https://github.com/bebatut)
+- [Mehmet Tekman](https://github.com/mtekman)
+- [Maria Doyle](https://github.com/mblue9)
+- [Pablo Moreno](https://github.com/pcm32)
+- [NH3 :)](https://github.com/nh3)
+- [Christophe Antoniewski](https://github.com/drosofff)
+- [Lea Bellenger](https://github.com/bellenger-l)
+- [Jelle Scholtalbers](https://github.com/scholtalbers)
+- [Charles Girardot](https://github.com/cgirardot)
+- [Nicola Soranzo](https://github.com/nsoranzo)
+- [Graham Etherington](https://github.com/ethering)
```

Comment:
@ethering are you ok that me mention you here?

Reply:
Yes, absolutely. That's great.